Dockerfile

```diff
@@ -13,4 +13,4 @@ COPY . .
 EXPOSE 8080
 ENV FLASK_ENV=development
 
-CMD bash -c "flask db upgrade && flask run --host 0.0.0.0 --port 8080"
+CMD ["gunicorn", "-w", "1", "-b", "0.0.0.0:8080", "app:create_app()"]
```

Comment:
This shouldn't be needed, or changed?

We don't want to mess with the Dockerfile, as that's used by both processes - it should not needed to be changed at all to allow it to run for paketo

Reply:
Removed